### ISSUE TITLE
`! Undefined control sequence. \@@magyar@captionfix ` with new latex format

### ISSUE THREAD

**Issue by u-fischer** [axelsommerfeldt/caption#7]
With the new latex format LaTeX2e <2018-04-01> patch level 2 (in Tl 2018 and miktex) the following document gives an error about an undefined `\@@magyar@captionfix`. 

Source of the problem is that caption relies on  a side effect of `\@ifundefined`  which has been removed in the new latex format : `\@ifundefined` not longer defines its argument. 

This means that this here from caption3.sty doesn't works:

~~~~~
  \@ifundefined{@@magyar@captionfix}\AtBeginDocument\@firstofone{%
    \g@addto@macro\@@magyar@captionfix{%
      \let\caption@lfmt@simple@magyar\caption@lfmt@simpleMagyar
      \let\caption@lfmt@simpleMagyar\caption@lfmt@default@magyar}}
~~~~~

If  ` \@@magyar@captionfix`  the  `\g@addto@macro\@@magyar@captionfix` will fail. 

~~~~~

\documentclass{beamer}
\usepackage{caption}
\begin{document}

\begin{frame}
Test
\end{frame}

\end{document}
~~~~~

**Comment by samcarter8** [comment]
related question: https://tex.stackexchange.com/questions/426088/texlive-pretest-2018-beamer-and-subfig-collide

**Comment by axelsommerfeldt** [comment]
...and uploaded to CTAN on 2018-04-15

**Comment by JohannesPfeifer** [comment]
@axelsommerfeldt Do you happen to know why only the `source`-subfolder at https://ftp.rrze.uni-erlangen.de/ctan/systems/win32/miktex/tm/packages/caption.tar.lzma contain the updated files while e.g. the `tex`-subfolder contains files from 2016? I experience this problem with every mirror I choose and thus cannot update the package. Instead, the Miktex console tells me that the package was correctly updated.

**Comment by u-fischer** [comment]
@JohannesPfeifer  Such packaging errors should be reported at the miktex issue tracker: https://github.com/MiKTeX/miktex-packaging/issues

**Comment by axelsommerfeldt** [comment]
@JohannesPfeifer No, I'm sorry. All I can do (and did) was creating a new version and upload it to CTAN. The MikTeX packages are build by the MikTeX maintainers, so I think the best one can do is asking them: https://github.com/MiKTeX/miktex-packaging/issues/

As workaround you could download the *.sty files from https://github.com/axelsommerfeldt/caption/tree/3.3/tex and copy them manually, replacing the existing ones.

**Comment by JohannesPfeifer** [comment]
@axelsommerfeldt Thank you very much for the quick reply. Replacing the files manually from the subfolder of the zipped repository worked like a charm.
@u-fischer Thanks for the pointer. I opened a ticket there.
